low-level/framework/utils/sysfs_interface.py

```diff
@@ -58,18 +59,23 @@ def initialize(self):
         except OSError as os_error:
             return os_error.errno
 
+    def get_sysfs_base_path(self):
```

Comment:
>low-level/framework/utils/sysfs_interface.py#L62
Method could be a function

Can mark as classmethod, with cls as first arg!? check link below

https://docs.quantifiedcode.com/python-anti-patterns/correctness/method_could_be_a_function.html

Reply:
> When a method is not preceded by the @staticmethod or @classmethod decorators and does not contain any references to the class or instance (via keywords like cls or self), Python raises the Method could be a function

But this method is already instantiated using self keyword as first arg.

Reply:
then decorator alone would help.

Reply:
okay. changed the instance method to class method